rlobodenko/05_HTML/new_1.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>  
+<HTML> 
+	<HEAD>
+	<TABLE border="0" width="700">
+	<THEAD>
+				<META charset="utf-8">
+			<TITLE>Формы</TITLE>
+				</THEAD>
+	</HEAD>  
+		<BODY> 
+			<TBODY>
+<TR>
+<TH><h2>Простая форма поиска с вариантами:</h2>
+<form method="get" action="http://sc.it-academy.by/fe/form.php">
+		<FIELDSET> 
+<LEGEND>Строка поиска</LEGEND> 
+<INPUT TYPE="text" NAME="search" MAXLENGTH="10" >
+<INPUT TYPE="submit" value="Искать"><br>
+<label>
+	<input type="radio" name="search_dist" value="site" checked>
+	на сайте
+</label>
+<label>
+	<input type="radio" name="search_dist" value="google">
+	в Google
+	</label><br>
+	<em></em>
+	</fieldset>
+</form></TH>
+<TH><hr>
+<h2>Простая форма поиска с вариантами:</h2>
+<form method="get" action="http://sc.it-academy.by/fe/form.php">
+			<FIELDSET> 
+<LEGEND>Строка поиска</LEGEND>
+<INPUT TYPE="text" NAME="search" MAXLENGTH="10" required 
+autofocus><span>*</span>
+	<input type="submit" value="Искать">
+<label>
+	<input type="radio" name="search_dist" value="site">
+	на сайте
+	</label>
+	<label>
+	<input type="radio" name="search_dist" value="google" checked>
+	в Google
+	</label><br>
+	</fieldset>
+</form></TH></TR>
+<TR>
+<TD><hr>
+<h2>Простая форма авторизации:</h2>
+<form method="get" action="http://sc.it-academy.by/fe/form.php">
+			<FIELDSET> 
+<LEGEND>Авторизуйтесь</LEGEND> 
+<INPUT TYPE="text" NAME="login" MAXLENGTH="10" required 
+autofocus><span>*</span>
+<label for="pass"> Логин </label>
+<input type="password" name="password"><span>*</span>
+<label for="pass"> Пароль </label>
+<label>  
+	<p><input type="checkbox" name="val_1" id="chbox1"> 
+	Оставаться в системе</p>
+	</label>
+<input type="submit" value="Войти"><br>
+	</fieldset>
+</form></TD>
+<TD><hr>
+<h2>Опрос(голосование):</h2>
+<form method="get" action="http://sc.it-academy.by/fe/form.php">
+			<FIELDSET> 
+<LEGEND>Как вам?</LEGEND> 
+<label>
+<p><input type="radio" name="date_voite" value="2013.12.31" id="Checkbox_Group1_0" checked> ничего</p>
```

Comment:
why in the here we have identical value and name? Does not matter what you choose, will be same